static/app/views/replays/detail/layout/utils.tsx

```diff
@@ -0,0 +1,15 @@
+import theme from 'sentry/utils/theme';
+import {LayoutModes} from 'sentry/views/replays/detail/layout';
+
+export const handleDefaultLayout = (): LayoutModes => {
+  const {innerWidth} = window;
+  const sidebarWidth = parseInt(theme.sidebar.expandedWidth, 10);
+
+  const windowsWidth = innerWidth - sidebarWidth;
+
+  if (windowsWidth < 1024) {
```

Comment:
So I was doing some testing, and personally I think after this spot the topbar layout is more comfortable than the sidebar layout - Of course this all based on personal taste

Reply:
Can we do something that considers the height as well? I'm really curious to see how the proportion of the window can affect things.

1024 seems reasonable on my laptop screen, but with different screen resolutions on external screens, ie: 2K, 4k I think it's not flexible enough to really change the output that people see.